cleansio/audio/audio_file.py

```diff
@@ -3,6 +3,7 @@
 import magic
 from pydub import AudioSegment
 from .convert import convert
+from .helper import *
```

Comment:
I heard that's bad practice to import everything, can you please change it to just what you need?